src/main/java/VASSAL/configure/PropertiesWindow.java

```diff
@@ -64,7 +65,7 @@ protected void initialize(final Configurable target, HelpWindow helpWindow) {
       if (Node.ELEMENT_NODE == child.getNodeType()) {
         // Cull Buildables from the state.
         try {
-          final Class<?> c = Class.forName(((Element)child).getTagName());
+          final Class<?> c = GameModule.getGameModule().getDataArchive().loadClass(((Element)child).getTagName(), false);
```

Comment:
Is there a reason we want `loadClass(String, boolean)` here instead of just `loadClass(String)`?

Reply:
(I suspect the answer is "no".)

Reply:
I'm pretty sure the answer is actually yes. loadClass(String) calls straight through into the bowels of SecureClassLoader and its parents. Whereas loadClass(String, boolean) is the version that is overridden in DataArchive... and is also where it has a specific flow designed to deal with checking for classes-we-actually-already-loaded (e.g. custom classes). 

And we definitely want the "resolve = false" end of the boolean, because all we're really doing here in this loop is "checking to see if the class legitimately exists" -- because what we're calling in with is the name of a supposed child class of some main class that we're editing, and we're deciding whether to cull its node(s) from the internal XML tree.

Reply:
Looks like loadClass(String) does simply call loadClass(String, boolean), which would call our overridden method. Still, I find calling the (String, boolean) version better, it shows explicitly what is called, if someone would jump into the method he would end up in our custom method right away and not have to navigate through internal Java methods. If for some reason the loadClass(String) would change the implementation and not call our method at all, we will not even notice that.

Reply:
My reason for wanting to avoid the boolean is that it's noise. Anyone modifying this code in the future will wonder what that `boolean` does, whether it should really be `false`, why this particular use of `loadClass` is not the same overload as the other ones in VASSAL... We could save future readers of the code all that trouble by using the overload without the `boolean`.

Reply:
Fixed